### PR TITLE
add sendSync to command bus

### PR DESCRIPTION
**Pull request by gtoselli**

### CODE REVIEW

File: packages/ddd-toolkit/src/command-bus/local-command-bus.ts [hunk 1]
```diff
@@ -16,12 +16,15 @@ export class LocalCommandBus implements ICommandBus {
         this.retryMechanism = new ExponentialBackoff(retryInitialDelay);
```
Comment (lucagiove) [inline packages/ddd-toolkit/src/command-bus/local-command-bus.ts:16]: retry is not used yet?

File: packages/ddd-toolkit/src/command-bus/local-command-bus.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { inspect } from 'util';
 export class LocalCommandBus implements ICommandBus {
     private readonly retryMechanism: IRetryMechanism;
 
-    private handlers: { [key: string]: ICommandHandler<ICommand<unknown>> } = {};
+    private handlers: { [key: string]: ICommandHandler<ICommand<unknown, unknown>> } = {};
 
     constructor(
```
Comment (lucagiove) [inline packages/ddd-toolkit/src/command-bus/local-command-bus.ts:11]: I would keep the same signature of local-event-bus either change the other

File: packages/ddd-toolkit/src/command-bus/local-command-bus.ts (hunk 1 above)
Comment (lucagiove) [reply packages/ddd-toolkit/src/command-bus/local-command-bus.ts:16]: we can remove it or add a TODO and a card in project

**Review by lucagiove — COMMENTED** [review]
all the rest seems fine

File: packages/ddd-toolkit/src/command-bus/local-command-bus.ts (hunk 1 above)
Comment (gtoselli) [reply packages/ddd-toolkit/src/command-bus/local-command-bus.ts:16]: @lucagiove Retry is used in the non-synchronous send. I don't understand, do you want to remove the non-synchronous send?

File: packages/ddd-toolkit/src/command-bus/local-command-bus.ts (hunk 2 above)
Comment (lucagiove) [reply packages/ddd-toolkit/src/command-bus/local-command-bus.ts:11]: @gtoselli my bad the diff hide the rest of the code I missed the usage of the retry. It's perfect sorry.

File: packages/ddd-toolkit/src/command-bus/local-command-bus.ts (hunk 1 above)
Comment (lucagiove) [reply packages/ddd-toolkit/src/command-bus/local-command-bus.ts:16]: @gtoselli my bad the diff hide the rest of the code I missed the usage of the retry. It's perfect sorry.

File: packages/ddd-toolkit/src/command-bus/local-command-bus.ts (hunk 2 above)
Comment (lucagiove) [reply packages/ddd-toolkit/src/command-bus/local-command-bus.ts:11]: I think I saw an old PR or I was drunk I referred to the LocalCommandBus constructor but probably bec I missed some code.

**Comment by lucagiove** [comment]
fine to merge for me 👌

**Pull request merged**